components/avatar/avatar.css

```diff
@@ -0,0 +1,77 @@
+@import url(/cssReset/reset.css);
+
+/* || Component Container */
+.component-container {
+  border: 1px solid lightgray;
+  border-radius: var(--small-border-radius);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  margin: 1.6rem auto;
+  max-width: 700px;
+}
+
+/* || Standard Avatar Component starts */
+
+.avatar-container {
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+}
+
+.avatar {
+  object-fit: cover;
+  height: 4.8rem;
+  width: 4.8rem;
+  margin: 1rem;
+  border-radius: var(--full-round-border-radius);
+}
+
+.avatar .avatar-image {
+  width: 100%;
+  height: 100%;
+  border-radius: var(--full-round-border-radius);
+}
+
+/* || Standard Avatar Component ends */
+
+/* || Different Sizes for Avatar starts */
+
+.avatar.avatar-xs {
+  height: 3.2rem;
+  width: 3.2rem;
+}
+.avatar.avatar-sm {
```

Comment:
Ankur, good work. Here you can add a line gap to maintain line gaps after each styles for classes.